docs/RELEASE.md

```diff
@@ -1,151 +1,26 @@
-# Release
+# Release Horreum
 
-## Building a base image
+## Creating a new stable branch
+This step is not required if you are going to perform a micro release from an already existing stable branch, e.g., `0.17.5`. 
 
-You can build a base image for the Horreum distribution with
+Open https://github.com/Hyperfoil/Horreum/actions/workflows/branch.yml and run the workflow. Select the `master` branch.
 
-```bash
-cd horreum-backend/
-docker build -f src/main/docker/Dockerfile.jvm.base -t quay.io/hyperfoil/horreum-base:latest .
-```
+## Perform Release
 
-Then it can be pushed to [quay.io](https://quay.io/) with
+Open https://github.com/Hyperfoil/Horreum/actions/workflows/release.yml and run the workflow. Select the previous branch created.
 
-```bash
-docker push quay.io/hyperfoil/horreum-base:latest
-```
+This workflow will take care of releasing all artifacts and container images on the appropriate registries:
+* Create a new GitHub tag following the semantic versioning (e.g., 0.17.0-SNAPSHOT → 0.17.0)
+* Push the maven artifacts to [Sonatype](https://s01.oss.sonatype.org/#nexus-search;quick~horreum)
+* Push the generated container image to [quay.io/hyperfoil/horreum](https://quay.io/repository/hyperfoil/horreum)
 
-if the proper credentials are in place.
+The only missing step is the creation of the GitHub release, a step that at the moment needs to be done manually using the GitHub UI. Consider using the “Generate release notes” feature to pre-populate the release notes automatically after selecting the correct tags.
```

Comment:
“Generate release notes” works well for point releases, but not for the first 0.x.0 release, the first time a new stable branch is released. This is due to us backporting commits to the previous stable branch, where features/fixes will have been backported to the 0.x-1.0 branch with a different commit hash.

We have not thought about how we can improve this, and atm for the first release from a new branch, we also have to manually remove the commits that had been backported from the release notes

Reply:
+1, unfortunately this seems a drawback of releasing from a stable branch where we cherry-pick commits (that, as you already highlighted, have different SHAs). 

Not sure whether we should add this clarification in the RELEASE.md doc 🫤

Reply:
IMHO, we should add what was mentioned. @johnaohara would you like to open a PR?